_template/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "",
+    "assets": [
+      {
+        "description": "The native staking token of EXAMPLE.",
+        "denom_units": [
+          {
+            "denom": "uexample",
+            "exponent": 0
+          },
+          {
+            "denom": "example",
+            "exponent": 6
+          }
+        ],
+        "base": "uexample",
+        "name": "Example",
+        "display": "exp",
```

Comment:
this display should match a denom_unit. You could change to 'example'. (It's something many PRs get wrong)